css/styles.css

```diff
@@ -408,6 +410,7 @@ header .menu-bars:hover {
 
 #contact h2 {
   color: #fff;
+  font-size: 40px;
 }
 
 #contact p {
```

Comment:
- [ ] The font size of the paragraph of `contact` is not fixed yet. You implemented `font-size: 20px` in the wrong class. Kindly implement `font-size: 20px` here as per the previous reviewers suggestion and to meet the given design guideline.

![image](https://user-images.githubusercontent.com/60083437/230739480-de421a34-2f68-4410-8513-c28f705dd090.png)